<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingsDialog</class>
 <widget class="QDialog" name="SettingsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>520</width>
    <height>604</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>520</width>
    <height>420</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Настройки</string>
  </property>
  <layout class="QVBoxLayout" name="mainLayout">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="generalTab">
      <attribute name="title">
       <string>Общие</string>
      </attribute>
      <layout class="QVBoxLayout" name="generalLayout">
       <item>
        <layout class="QFormLayout" name="languageFormLayout">
         <item row="0" column="0">
          <widget class="QLabel" name="languageLabel">
           <property name="text">
            <string>Язык интерфейса:</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QComboBox" name="languageComboBox">
           <property name="toolTip">
            <string>Применится после перезапуска приложения</string>
           </property>
           <item>
            <property name="text">
             <string>По умолчанию (система)</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>English</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Русский</string>
            </property>
           </item>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QFormLayout" name="logPathFormLayout">
         <item row="0" column="1">
          <widget class="QWidget" name="logPathWidget" native="true">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
             <horstretch>1</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <layout class="QHBoxLayout" name="logPathLayout">
            <item>
             <widget class="QLineEdit" name="logPathEdit">
              <property name="toolTip">
               <string>Каталог для файлов логов. Имя файла: easypos_дата_время_хост.txt</string>
              </property>
              <property name="placeholderText">
               <string>по умолчанию</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="logPathBrowseButton">
              <property name="toolTip">
               <string>Выбрать каталог для сохранения логов</string>
              </property>
              <property name="text">
               <string>Обзор…</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="logPathLabel">
           <property name="toolTip">
            <string>Пустой = по умолчанию. При каждом запуске создаётся новый файл (дата, время, хост сервера). Применится после перезапуска.</string>
           </property>
           <property name="text">
            <string>Каталог логов:</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QCheckBox" name="printAfterPayCheckBox">
         <property name="toolTip">
          <string>Автоматически открывать диалог печати после закрытия чека</string>
         </property>
         <property name="text">
          <string>Печатать чек после оплаты</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Orientation::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="databaseTab">
      <attribute name="title">
       <string>База данных</string>
      </attribute>
      <layout class="QVBoxLayout" name="databaseLayout">
       <item>
        <widget class="QLabel" name="dbDescriptionLabel">
         <property name="text">
          <string>Параметры подключения к PostgreSQL (хост, порт, имя базы, пользователь, пароль). Изменение вступает в силу при следующем подключении.</string>
         </property>
         <property name="wordWrap">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="dbCurrentLabel">
         <property name="text">
          <string>Текущее подключение: —</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="dbSettingsButton">
         <property name="text">
          <string>Настроить подключение к БД...</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer2">
         <property name="orientation">
          <enum>Qt::Orientation::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="usersRolesTab">
      <attribute name="title">
       <string>Пользователи и роли</string>
      </attribute>
      <layout class="QVBoxLayout" name="usersRolesLayout">
       <item>
        <widget class="QLabel" name="rolesLabel">
         <property name="font">
          <font>
           <pointsize>10</pointsize>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Роли</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTableWidget" name="rolesTable">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>100</height>
          </size>
         </property>
         <property name="selectionMode">
          <enum>QAbstractItemView::SelectionMode::SingleSelection</enum>
         </property>
         <property name="selectionBehavior">
          <enum>QAbstractItemView::SelectionBehavior::SelectRows</enum>
         </property>
         <column>
          <property name="text">
           <string>Название</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Уровень</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Заблокирована</string>
          </property>
         </column>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="rolesButtonsLayout">
         <item>
          <widget class="QPushButton" name="rolesAddBtn">
           <property name="text">
            <string>Добавить роль</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="rolesEditBtn">
           <property name="text">
            <string>Изменить</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="rolesDeleteBtn">
           <property name="text">
            <string>Удалить</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Orientation::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="usersHeaderLayout">
         <item>
          <widget class="QLabel" name="usersLabel">
           <property name="font">
            <font>
             <pointsize>10</pointsize>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>Пользователи</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="showDeletedUsersCheckBox">
           <property name="text">
            <string>Показать удалённых</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QTableWidget" name="usersTable">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>100</height>
          </size>
         </property>
         <property name="selectionMode">
          <enum>QAbstractItemView::SelectionMode::SingleSelection</enum>
         </property>
         <property name="selectionBehavior">
          <enum>QAbstractItemView::SelectionBehavior::SelectRows</enum>
         </property>
         <column>
          <property name="text">
           <string>Имя</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Роль</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Бейдж</string>
          </property>
         </column>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="usersButtonsLayout">
         <item>
          <widget class="QPushButton" name="usersAddBtn">
           <property name="text">
            <string>Добавить пользователя</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="usersEditBtn">
           <property name="text">
            <string>Изменить</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="usersDeleteBtn">
           <property name="text">
            <string>Удалить</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="usersRestoreBtn">
           <property name="toolTip">
            <string>Показать удалённых и восстановить выбранного</string>
           </property>
           <property name="text">
            <string>Восстановить</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer2">
           <property name="orientation">
            <enum>Qt::Orientation::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::StandardButton::Cancel|QDialogButtonBox::StandardButton::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
